### PR TITLE
Create io.github.prateekmedia.flutube

### PR DESCRIPTION
**Pull request by prateekmedia**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by prateekmedia** [comment]
bot, build io.github.prateekmedia.flutube

**Comment by prateekmedia** [comment]
bot, build io.github.prateekmedia.flutube

**Comment by prateekmedia** [comment]
bot, build io.github.prateekmedia.flutube

**Comment by prateekmedia** [comment]
bot, build io.github.prateekmedia.flutube

**Comment by prateekmedia** [comment]
bot, build io.github.prateekmedia.flutube

File: io.github.prateekmedia.flutube.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+command: flutube
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --share=network
+  - --device=dri
+  - --filesystem=~/FluTube:create
```
Comment (barthalion) [inline io.github.prateekmedia.flutube.yml:17]: Any reason it doesn't use `xdg-download` for storing files?

Comment (prateekmedia) [reply io.github.prateekmedia.flutube.yml:17]: No concrete reason, just to keep youtube downloaded music and videos seperate from normal downloads.

Comment (barthalion) [reply io.github.prateekmedia.flutube.yml:17]: I think polluting home directory is generally considered bad practice. Common downloads directory exists exactly for that use case.

Comment (prateekmedia) [reply io.github.prateekmedia.flutube.yml:17]: Okay, I will then create a sub-directory inside `Downloads` named `FluTube` to keep them seperate.

**Comment by prateekmedia** [comment]
bot, build io.github.prateekmedia.flutube

**Comment by barthalion** [comment]
bot, build io.github.prateekmedia.flutube

**Comment by prateekmedia** [comment]
bot, build io.github.prateekmedia.flutube

**Comment by prateekmedia** [comment]
@barthalion Do you know what's the correct way to check it for github releases, It seems it is still happening.

**Comment by prateekmedia** [comment]
bot, build io.github.prateekmedia.flutube

**Comment by barthalion** [comment]
```
[ERROR:flutter/lib/ui/ui_dart_state.cc(209)] Unhandled Exception: SocketException: Connection failed (OS Error: No such file or directory, errno = 2), address = /var/run/dbus/system_bus_socket, port = 0
#0      _NativeSocket.startConnect (dart:io-patch/socket_patch.dart:677)
#1      _NativeSocket.connect (dart:io-patch/socket_patch.dart:914)
#2      _RawSocket.connect (dart:io-patch/socket_patch.dart:1668)
#3      RawSocket.connect (dart:io-patch/socket_patch.dart:21)
#4      Socket._connect (dart:io-patch/socket_patch.dart:1883)
#5      Socket.connect (dart:io/socket.dart:724)
#6      DBusClient._openSocket (package:dbus/src/dbus_client.dart:721)
#7      DBusClient._connect (package:dbus/src/dbus_client.dart:741)
#8      DBusClient.callMethod (package:dbus/src/dbus_client.dart:563)
#9      DBusClient.getNameOwner (package:dbus/src/dbus_client.dart:401)
#10     DBusClient._findUniqueName (package:dbus/src/dbus_client.dart:580)
#11     DBusSignalStream._onListen (package:dbus/src/dbus_client.dart:132)
#12     _runGuarded (dart:async/stream_controller.dart:773)
#13     _StreamController._subscribe.<anonymous closure> (dart:async/stream_controller.dart:651)
#14     _BufferingStreamSubscription._guardCallback (dart:async/stream_impl.dart:417)
#15     _StreamController._subscribe (dart:async/stream_controller.dart:650)
#16     _ControllerStream._createSubscription (dart:async/stream_controller.dart:786)
#17     _StreamImpl.listen (dart:async/stream_impl.dart:473)
#18     DBusSignalStream.listen (package:dbus/src/dbus_client.dart:126)
#19     new _ForwardingStreamSubscription (dart:async/stream_pipe.dart:114)
#20     _ForwardingStream._createSubscription (dart:async/stream_pipe.dart:86)
#21     _ForwardingStream.listen (dart:async/stream_pipe.dart:81)
#22     NetworkManagerClient.connect (package:nm/src/network_manager_client.dart:2216)
#23     ConnectivityLinux.checkConnectivity (package:connectivity_plus_linux/src/connectivity_real.dart:19)
#24     Connectivity.checkConnectivity (package:connectivity_plus/connectivity_plus.dart:66)
#25     InternetConnectivity.networkStatusService (package:flutube/controller/internet_connectivity.dart:30)
#26     main (package:flutube/main.dart:22)
#27     main (file:///home/runner/work/flutube/flutube/.dart_tool/flutter_build/generated_main.dart)
#28     _runMainZoned.<anonymous closure>.<anonymous closure> (dart:ui/hooks.dart:143)
#29     _rootRun (dart:async/zone.dart:1428)
#30     _CustomZone.run (dart:async/zone.dart:1328)
#31     _runZoned (dart:async/zone.dart:1863)
#32     runZonedGuarded (dart:async/zone.dart:1851)
#33     _runMainZoned.<anonymous closure> (dart:ui/hooks.dart:141)
#34     _delayEntrypointInvocation.<anonymous closure> (dart:isolate-patch/isolate_patch.dart:287)
#35     _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:187)
```

Looks like it misses some `--talk-name` to detect network connectivity? But it seems to work fine, so feel free to fix it later.

/merge

**Pull request closed without merging**